tests/data/power_flow/automatic-tap-regulator/auto-tap-changer-meshed-any-max-iter/params.json

```
@@ -1,6 +1,6 @@
 {
   "calculation_method": "newton_raphson",
-  "tap_changing_strategy": "any_valid_tap",
+  "tap_changing_strategy": "fast_any_tap",
```

Do we support the multiple tap_changing_strategy? 
in that case
```suggestion
  "tap_changing_strategy": ["any_valid_tap", "fast_any_tap"],
```